docs/contracts.rst

```diff
@@ -24,8 +24,8 @@ Creating contracts programatically on Ethereum is best done via using the JavaSc
 As of today it has a method called `web3.eth.Contract <https://web3js.readthedocs.io/en/1.0/web3-eth-contract.html#new-contract>`_
 to facilitate contract creation.
 
-When a contract is created, its constructor (a function declared with the
-``constructor`` keyword) is executed once.
+When a contract is created, its constructor (a function declared with the ``constructor`` keyword) is executed once, making it useful if you want to execute code when the contract is first mined. See `replicator.sol <https://github.com/fivedogit/solidity-baby-steps/blob/master/contracts/50_replicator.sol>`_ for an example.
```

Comment:
I think we should not link the contracts by fivedogit, they are really outdated. Also, I think the grammar of the second part of the sentence could be simplified (`allowing to execute code when the contract is created`). But actually - isn't that what the first part of the sentence already says?